src/main/kotlin/com/workos/usermanagement/models/InvitationEvent.kt

```diff
@@ -0,0 +1,49 @@
+package com.workos.usermanagement.models
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.workos.usermanagement.types.InvitationStateEnumType
+
+/**
+ * An email invitation allows the recipient to sign up for your app and join a
+ * specific organization. When an invitation is accepted, a user and a
+ * corresponding organization membership are created.
+ *
+ * @param id The unique ID of the invitation.
+ * @param email The email address of the user.
+ * @param state The state of the invitation (see enum values in [InvitationStateEnumType]).
+ * @param acceptedAt The timestamp when the invitation was accepted.
+ * @param revokedAt The timestamp when the invitation was revoked.
+ * @param expiresAt The timestamp when the invitation will expire.
+ * @param organizationId The ID of the organization.
+ * @param createdAt The timestamp when the invitation was created.
+ * @param updatedAt The timestamp when the invitation was last updated.
+ */
+data class InvitationEvent @JsonCreator constructor(
```

Comment:
Confirming my understanding - fetching invitations from the API contains the `acceptInvitationUrl` key, but the Invitation object returned for events does not, is that correct?

Reply:
The API response object contains the `token` and the `acceptInvitationUrl` but the event does not, as those are secrets, which we don't want to expose in events. More context in the decision log here: https://workos.quip.com/lLKGA1TcBTrV/User-Management-Configurable-Emails-Decision-Log#temp:C:GRO1a7c777038db4dea886af97c8